common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -260,4 +265,20 @@ public static Date RFC3339StringToDate(@NonNull String dateStr) throws ParseExce
         return RFC3339DateFormat.parse(dateStr);
     }
 
+    /**
+     * Create the Hash string of the message.
+     *
+     * @param msg String
+     * @return String in Hash
+     * @throws NoSuchAlgorithmException throws if no such algorithm.
+     */
+    public static String createHash(final String msg) throws NoSuchAlgorithmException {
```

Comment:
nit: can you kill its counterpart in StringExtension?

Reply:
I tried...but that one for some reason also throws an `UnsupportedEncodingException`. I don't know why because it is the same code...it seems when the code is written in an Android module then it uses the java in Android SDK for compilation and that throws an UnsupportedEncodingException when calling `getBytes` on a String...that `createHash` method was being used in a lot of places and the `UnsupportedEncodingException` was being thrown up the hierarchy and I didn't want to touch all that now...so I left it alone. 😁